lib/internal/process/pre_execution.js

```diff
@@ -550,6 +551,23 @@ function initializePermission() {
         'It could invalidate the permission model.', 'SecurityWarning');
       }
     }
+    const warnCommaFlags = [
+      '--allow-fs-read',
+      '--allow-fs-write',
+    ];
+    for (const flag of warnCommaFlags) {
+      const value = getOptionValue(flag);
+      if (value.length === 1 && value[0].includes(',')) {
+        process.emitWarning(
+          `the ${flag} CLI flag has recently changed. ` +
+        'Passing a comma separated list of files is no longer valid. ' +
+        `Each file/directory should have its own ${flag} call. ` +
+        `Example: ${flag}=/folder1/ ${flag}=/folder2/. ` +
+        'If the provided value is a single path with commas' +
+        'this warning can be safely ignored', 'Warning',
```

Comment:
```suggestion
        `Each file/directory should have its own ${flag} call. ` +
        `Example: ${flag}=/folder1/ ${flag}=/folder2/. ` +
        'If the provided value is a single path with commas' +
        'this warning can be safely ignored', 'Warning',
```

I think at this point we should just point them to the permission docs

Reply:
It doesn't let me commit, I'll change it myself